ppb/systems/inputs.py

```diff
@@ -3,11 +3,11 @@
 import pygame
 
 import ppb.buttons as buttons
-from ppb import Vector
-from ppb import events
-from ppb import keycodes as keys
-from ppb.systems import System  # TODO: Be aware of circular imports
-from ppb.systems import default_resolution
+from ppb_vector import Vector
+import ppb.events as events
+import ppb.keycodes as keys
+from ppb.systems.base import System
+from ppb.systems.renderer import DEFAULT_RESOLUTION
```

Comment:
Not blocking this, but we should probably move shared defaults to their own file somewhere.

Reply:
Possibly? If we had more than one?

I grouped it like this because resolution felt like a renderer concern, and the renderer is responsible for window management currently?